portfolio/templates/main/entry.html

```diff
@@ -67,7 +67,13 @@ <h1 class="align-self-center" id="maincontent">{{page.title}}</h1>
                 <dl class="text-center m-0">
                 {% with partners=page.partners.all %}
                     {% if partners %}
-                    <dt class="field-label">Partners</dt>
+                    <dt class="field-label">
+                    {% if partners.count > 1 %}
```

Comment:
Note: the `pluralize` templatetag will also handle the case where there are 0 partners. https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#pluralize In reality, you won't have to deal with this case, so what you've done here is all good.